uploader.json

```diff
@@ -1286,6 +1286,11 @@
             "command": "/usr/bin/yum -C repolist",
             "pattern": [],
             "symbolic_name": "yum_repolist"
+        },
+        {
+            "command": "/usr/bin/abrt status --bare=True",
+            "pattern": [],
+            "symbolic_name": "abrt_status_bare"
```

Comment:
@ernestask these entries should be in alpha sorted order by symbolic name.  Please move this entry so that it will be in the correct order.

Reply:
I have to say, though, there already are violations of that rule. azure_instance_type comes before auditctl_status.